lib/image-stream/handlers.js

```diff
@@ -50,6 +52,8 @@ module.exports = {
   'application/x-bzip2': (file, options) => {
     return Bluebird.props({
       path: file,
+      archiveExtension: path.extname(file),
```

Comment:
I guess it depends if it makes statistics / results easier to generate / sort / filter, if the `archiveExtension` is 'automatically' there, without having to manually extract it from the filename?